query_classifier/qc_sqlite/sqlite-src-3110100/test/index3.test

```diff
@@ -42,7 +42,7 @@ integrity_check index3-1.4
 # Backwards compatibility test:
 #
 # Verify that CREATE INDEX statements that use strings instead of 
-# identifiers for the the column names continue to work correctly.
+# identifiers for the column names continue to work correctly.
```

Comment:
Sorry, I excluded this content in my initial work, but when I rebased I wasn't careful to ensure that my filters were applied when I ran some fixups. I'll drop this.